geoutils/georaster.py

```diff
@@ -1250,41 +1254,67 @@ def interp_points(self,pts,nsize=1,mode='linear',band=1):
         assert mode in ['mean', 'linear', 'cubic', 'quintic',
                         'nearest'], "mode must be mean, linear, cubic, quintic or nearest."
 
-        rpts = []
+        # get coordinates
+        x, y = list(zip(*pts))
 
-        #TODO: might need to check if coordinates are center or point in the metadata here...
+        # if those are in latlon, convert to Raster crs
+        if input_latlon:
+            init_crs = pyproj.CRS(4326)
```

Comment:
If we don't, we should definitely have a small function for this type of operation in projtools !